recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -81,6 +77,7 @@ EXTRA_OECMAKE = "-DTARGET_PLATFORM=${TARGET_PLATFORM} \
                  -DUSE_USER_TMP_PATH=0 \
                  -DCMAKE_INSTALL_PREFIX:PATH=${INSTALL_PREFIX} \
                  -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+		 ${@bb.utils.contains('PACKAGECONFIG', 'bwl-nl80211', '-DBWL_LIBS=wpa_client', '', d)} \
```

Comment:
Isn't indentation wrong here?

Wouldn't it be simpler to add this to the `PACKAGECONFIG[bwl-nl80211]` line above?

BTW, very nice workaround! I had expected we'd have to do more hacks in the prplmesh CMakeLists.txt to make this work, but this is more elegant. Still, maybe add a comment to explain why we need to set BWL_LIBS explicitly - it's an "invisible" CMake variable so somewhat exotic...